samples/azure-quantum/chemistry/README.md

```diff
@@ -0,0 +1,39 @@
+---
+page_type: sample
+author: guenp
+description: Estimate the molecular ground state energy of hydrogen using the Azure Quantum service
+ms.author: guenp@microsoft.com
+ms.date: 05/18/2021
+languages:
+- qsharp
+- python
+products:
+- qdk
+- azure-quantum
+---
+
+# Simple molecular energy estimation with the Azure Quantum service
+
+This sample demonstrates how to use Q#, Python and the Azure Quantum service together to estimate the ground state of a molecule, implemented as two Jupyter notebooks.
+
+## Running the sample
+
+To run the sample, first make sure to install all [prerequisites](https://docs.microsoft.com/en-us/azure/quantum/install-python-qdk?tabs=tabid-conda) and start a Jupyter notebook with the following command:
+
+```cmd
+jupyter notebook
+```
+
+Open the notebooks ([QuantumPhaseEstimation/Molecule.ipynb](QuantumPhaseEstimation/Molecule.ipynb) or [Hamiltonian/Molecule.ipynb](Hamiltonian/Molecule.ipynb)) and execute the cells (shift + enter) to run the sample.
```

Comment:
Should we mention that the user needs AQ registration before running the second sample?

Reply:
Added!